### ISSUE TITLE
New inbound SMS start new Posts AND appear on existing Posts' conversations with Author section

### ISSUE THREAD

**Issue by willdoran** [ushahidi/platform#2343]
### Expected behaviour
- The conversation with an Author does not spawn new Posts each time the Author responds
### Actual behaviour
- Each time the Author responds a new Post is created
### Steps to reproduce the behaviour/error
- Requires: Datasource SMS setup
- [ ] Receive an inbound SMS
- [ ] Go to the Post created for this inbound SMS
- [ ] Using the Conversation with Author section
- [ ] Send a reply to the Author

##### Where
- [x] [Local setup with Vagrant ](https://www.ushahidi.com/support/install-ushahidi#installing-for-development)
- [x] [Local setup from platform-release ](https://www.ushahidi.com/support/install-ushahidi#installing-the-latest-release)
- [x] Ushahidi.io / SaaS solution 
- [x] Ushahidi's QA environment
- [x] Other (explain):

**Comment by jshorland** [comment]
This should be fixed within the SMS targeted surveys

**Comment by rowasc** [comment]
First sent two messages to /smssync
<img width="1093" alt="screen shot 2018-02-28 at 2 45 33 pm" src="https://user-images.githubusercontent.com/2434401/36803595-c2816c76-1c96-11e8-9e9b-a8e85d472f63.png">
Then answered and sent another message and this was the result
<img width="1593" alt="screen shot 2018-02-28 at 2 59 48 pm" src="https://user-images.githubusercontent.com/2434401/36804698-22e93cee-1c9a-11e8-9ca2-c0540641bf50.png">

Q1 : So if I'm in the data view, for instance, it would be shown under the conversation with author but not in the sidebar of posts. Right? 
Q2: not sure how Eve would test this? directly with SMS or should we send her a postman set up for this?

**Comment by willdoran** [comment]
@rowasc 
1. This is already covered by the SMS Feature issue #2532
2. We have test accounts for both nexmo and twillio, we should have one for frontlinesms those have numbers that are text able to text

**Comment by willdoran** [comment]
Closing in favour of #2532

**Comment by rowasc** [comment]
makes sense thanks
